libs/core/src/lib/table/table.component.scss

```diff
@@ -1 +1,524 @@
-@import '~fundamental-styles/dist/table';
+.fd-table__popover--custom {
+    width: 100%;
+}
+
+.fd-table-hidden {
+    display: none;
+}
+//@import '~fundamental-styles/dist/table';
+
+/*!
+ * Fundamental Library Styles v0.11.0-rc.7
+ * Copyright (c) 2020 SAP SE or an SAP affiliate company.
+ * Licensed under Apache License 2.0 (https://github.com/SAP/fundamental-styles/blob/master/LICENSE)
+ */
+@charset "UTF-8";
+/*!
+.fd-table(--compact, --condensed, --no-horizontal-borders, --no-vertical-borders)
+    thead(.fd-table__header(--no-horizontal-borders, --no-vertical-borders))
+    tbody(.fd-table__body(--no-horizontal-borders, --no-vertical-borders))
+    tfoot(.fd-table__footer)
+        tr(.fd-table__row)([aria-selected])
+          td(.fd-table__cell(--status-indicator, --checkbox, --fit-content)) + semantic
+*/
+.fd-table {
```

Comment:
Could you take latest styles there? pop-in table is fixed on current styles master.